src/testers.jl

```diff
@@ -59,6 +77,35 @@ function _make_fdm_call(fdm, f, ȳ, xs, ignores)
     return (args...,)
 end
 
+"""
+    _make_jvp_call(fdm, f, xs, ẋs, ignores)
+
+Call `FiniteDifferences.jvp`, with the option to ignore certain `xs`.
+
+# Arguments
+- `fdm::FiniteDifferenceMethod`: How to numerically differentiate `f`.
+- `f`: The function to differentiate.
+- `xs`: Inputs to `f`, such that `y = f(xs...)`.
+- `ẋs`: Derivative of `xs` w.r.t. some real number `t`.
+- `ignores`: Collection of `Bool`s, the same length as `xs` and `ẋs`.
+   If `ignores[i] === true`, then `ẋs[i]` is ignored for derivative estimation.
+
+# Returns
+- `∂Ω`: Derivative of output w.r.t. `t` estimated by finite differencing.
```

Comment:
Should we start calling it `Ω̇`, or would that be confusing because it is being estimated via finite differencing?
```suggestion
- `Ω̇`: Derivative of output w.r.t. `t` estimated by finite differencing.
```